Source/Transport/Simple.H

```diff
@@ -160,14 +158,18 @@ struct BinaryDiff
                                tparm->fitdbin[four_idx_ij + 1] * logT[0] +
                                tparm->fitdbin[four_idx_ij + 2] * logT[1] +
                                tparm->fitdbin[four_idx_ij + 3] * logT[2];
-
         dbintemp = std::exp(-dbintemp);
         Ddiag[i] += Xloc[j] * dbintemp;
         Ddiag[j] += Xloc[i] * dbintemp;
       }
     }
     for (int i = 0; i < NUM_SPECIES; ++i) {
```

Comment:
I think we could merge this for loop into the previous one, to avoid doing an extra N^2 loop, but also remove the if statement, i.e. 
1. we add term1=0 at the start of the loop
2. in the first nested loop (j < i) we add term1 += yloc[j]
3. Add another loop j=i+1, j < NUM_SPECIES which just adds the remaining part of term1
4. Construct Ddiag in the i loop.